protocol/op-contracts-manager-single-release-redesign.md

```diff
@@ -0,0 +1,83 @@
+# Purpose
+
+The original OPContractsManager (OPCM) design, outlined [here](./op-contracts-manager-arch.md), defined a singleton proxied contract that deployed the L1 contracts for standard OP Stack chains in a single transaction.
```

Comment:
This reminds me that we will also need to update the spec once this design doc has consensus: https://specs.optimism.io/experimental/op-contracts-manager.html

Reply:
Tracked in Figma.